### DISCUSSION TITLE
Remove file from downloadUrl

### DISCUSSION THREAD

**Discussion by codeoffconduct** [firebase/flutterfire#4392] [Feature request]
I can't find a clear way to remove a file from a downloadUrl. I have seen many people online asking how to do this. 
Could you please add this to the roadmap?

Thanks

**Comment by shatanikmahanty** [comment]
This flutter code worked for me `await storage().refFromURL("Your link goes here").delete();` [This code is web-only as storage library is giving unimplemented for web. For other supported platforms use below code]
I imported : 
  `firebase: ^7.3.2`

If you are writing using FirebaseStorage you should write :
    `await FirebaseStorage.instance.refFromURL(model.bannerLink).delete();`
import : 
`firebase_storage: ^5.2.0`
